bedrock/exp/urls.py

```diff
@@ -16,7 +16,6 @@
     page('opt-out', 'exp/opt-out.html'),
     page('firefox', 'exp/firefox/index.html', ftl_files=['firefox/home']),
     url(r'^firefox/new/$', views.new, name='exp.firefox.new'),
-    page('firefox/mobile', 'exp/firefox/mobile.html', ftl_files=['firefox/mobile']),
```

Comment:
Is it worth adding a redirect for this URL, incase there are external links to it?